dask/array/tests/test_linalg.py

```diff
@@ -449,35 +449,22 @@ def test_dask_svd_self_consistent(m, n):
         assert d_e.dtype == e.dtype
 
 
-@pytest.mark.slow
 def test_svd_compressed():
-    m, n = 2000, 250
-    r = 10
-    np.random.seed(4321)
-    mat1 = np.random.randn(m, r)
-    mat2 = np.random.randn(r, n)
-    mat = mat1.dot(mat2)
-    data = da.from_array(mat, chunks=(500, 50))
-
-    u, s, vt = svd_compressed(data, r, seed=4321, n_power_iter=2)
-
-    usvt = da.dot(u, da.dot(da.diag(s), vt))
-
-    tol = 0.2
-    assert_eq(
-        da.linalg.norm(usvt), np.linalg.norm(mat), rtol=tol, atol=tol
-    )  # average accuracy check
+    m, n = 100, 50
+    r = 5
+    a = da.random.random((m, n), chunks=(m, n))
```

Comment:
Spectrum is better suited for this type of testing.

Reply:
👍